### PR TITLE
[SiVal] Enable spi_host_tpm test for SiVal  nightly

### PR DESCRIPTION
**Pull request by engdoreis**
This PR move the tpm test to run on the CW340 and create a job to run all the SiVal cw340 tests.

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -86,6 +86,70 @@ jobs:
           BUCKET_PATH=$GS_PATH/job/${{ github.job }}/branch/${{ inputs.branch || 'earlgrey_1.0.0'}}/$(date +%Y-%m-%d-%H%M%S)_${BAZEL_TEST_RESULTS}
           gcloud storage cp $BAZEL_TEST_RESULTS "$BUCKET_PATH"
 
+  fpga_cw340_sival_nightly:
+    name: FPGA CW310 SiVal tests
```
Comment (nbdd0121) [inline .github/workflows/nightly.yml:90]:
```suggestion
    name: FPGA CW340 SiVal tests
```

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -86,6 +86,70 @@ jobs:
...
+    runs-on: [ubuntu-22.04-fpga, cw310]
```
Comment (nbdd0121) [inline .github/workflows/nightly.yml:91]:
```suggestion
    runs-on: [ubuntu-22.04-fpga, cw340]
```

Comment (engdoreis) [reply .github/workflows/nightly.yml]: Thanks for catch that!

**Review by luismarques — APPROVED** [review]

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (engdoreis) [reply .github/workflows/nightly.yml]: Fixed

**Pull request merged**